src/buffer/geometry-buffer.ts

```diff
@@ -152,14 +152,16 @@ abstract class GeometryBuffer extends MeshBuffer {
           position[ i3 ], position[ i3 + 1 ], position[ i3 + 2 ]
         )
         this.applyPositionTransform(matrix, i, i3)
-        applyMatrix4toVector3array(matrix.elements, transformedGeoPosition)
```

Comment:
Same as in surface/volume.ts: force the typing here with a `this.matrix.elements as unknown as Float32Array`
(and same l:163)